mscore/plugin/api/playevent.h

```diff
@@ -0,0 +1,118 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2019 Werner Schweer and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2
+//  as published by the Free Software Foundation and appearing in
+//  the file LICENCE.GPL
+//=============================================================================
+
+#ifndef __PLUGIN_API_PLAYEVENT_H__
+#define __PLUGIN_API_PLAYEVENT_H__
+
+#include "libmscore/noteevent.h"
+#include "elements.h"
+
+namespace Ms {
+namespace PluginAPI {
+
+class Note;
+
+//---------------------------------------------------------
+//   PlayEvent
+//    Wrapper class for internal Ms::NoteEvent
+//
+//   This is based on the wrapper in excerpt.h.
+//---------------------------------------------------------
+
+class PlayEvent : public QObject {
+      Q_OBJECT
+      /// The relative pitch of the event.
+      /// This is added to the parent note's actual pitch.
+      /// \since MuseScore 3.3
+      Q_PROPERTY(int pitch READ pitch WRITE setPitch)
+      /// Time to turn on the note event.
```

Comment:
`ontime`, `len` and `offtime` properties could have more details documented regarding what their values mean. It [follows](https://github.com/musescore/MuseScore/blob/6844e45ffa109138631b1db174e2cf59cd4da349/libmscore/noteevent.h#L27) from the libmscore code that these values are expressed in `1/1000` of the nominal note length but this is not reflected in the current version of documentation to these properties.

Reply:
fixed.